pypeit/data/extinction/README

```diff
@@ -1,7 +1,7 @@
 # Airmass Extinction files
 #   Generally grabbed from the Internet.  Buyer beware
     File             Lon         Lat      Elevation  
-ctioextinct.dat     70.8150    -30.1653   2215.0
+ctioextinct.dat     -70.8150    -30.1653   2215.0
```

Comment:
return back.   Am pretty sure these aren't used.
If so, then we need to think more carefully.

Reply:
I am copying @profxj's comment. The west should be positive, please change it back to positive value for the longitude. We do use the extinction file for the UVIS flux calibration if the coordinates of the observatory is within 5 degrees from that of the extinction file.